sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md

```diff
@@ -603,70 +603,86 @@ manager, with `__aenter__`, `__aexit__`, and `close` methods.
 
 ## Long-running operation (LRO) customization
 
- API operations that can take longer to complete are often represented as long-running operations (LRO). `azure-core` provides `LROPoller` (and `AsyncLROPoller`) protocols
- that give users access to the LRO and expose methods to interact with them such as waiting until the operation reaches a terminal state, checking its status, or
- providing a callback to do work on the final result when it is ready. If the LRO follows the [Azure REST API guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs), 
+ Operations that are started by an initial call, then need to be monitored for status until completion are often represented as long-running operations (LRO).
+ The `azure-core` library provides the [LROPoller][lro_poller] (and [AsyncLROPoller`][async_lro_poller]) protocols
+ that expose methods to interact with the LRO such as waiting until the operation reaches a terminal state, checking its status, or
+ providing a callback to do work on the final result when it is ready. If the LRO follows the 
+ [Azure REST API guidelines][rest_api_guidelines_lro], 
  it's likely that the generated client library code should _just work_. 
  In cases where the LRO diverges from the guidelines, you may need to customize your code to achieve the desired scenario.
 
- There are 3 options to customize the logic for our LROs.
+ There are 3 options to customize the logic for LROs.
 
- 1) [LRO algorithm/strategies - OperationResourcePolling, LocationPolling, StatusCheckPolling](#lro-algorithmstrategies---operationresourcepolling-locationpolling-statuscheckpolling)
+ 1) [Polling strategy - OperationResourcePolling, LocationPolling, StatusCheckPolling](#polling-strategy---operationresourcepolling-locationpolling-statuscheckpolling)
     - You need to customize the polling strategy
  2) [Polling method - LROBasePolling/AsyncLROBasePolling](#polling-method---lrobasepollingasynclrobasepolling)
     - You need to customize the polling loop
- 3) [Poller - LROPoller/AsyncLROPoller](#poller---lropollerasynclropoller)
+ 3) [Poller API - LROPoller/AsyncLROPoller](#poller-api---lropollerasynclropoller)
     - You need to customize the public interface of the poller
 
+The poller API is what the user uses to interact with the LRO. Internally, the poller uses the polling method to run the polling
+loop which makes calls to the status monitor, controls delay, and determines when the LRO has reached a terminal state.
+The polling method is supplied a polling strategy which defines where and how to the get the status from the status monitor.
```

Comment:
Rather than "is supplied a", maybe just "uses"? e.g. something like:
`The "polling method" uses a "polling strategy" to determine how to extract the status information from the responses returned by the status monitoring.`